driver-core/src/main/com/mongodb/internal/async/client/AsyncMongoClient.java

```diff
@@ -233,4 +236,19 @@
     <TResult> AsyncChangeStreamIterable<TResult> watch(AsyncClientSession clientSession, List<? extends Bson> pipeline,
                                                        Class<TResult> resultClass);
 
+    /**
```

Comment:
This method is now used in the test runner, so it was need in SyncMongoClient, and therefore, here as well.